iw_parse.py

```diff
@@ -108,14 +132,18 @@ def matching_line(lines, keyword):
 
 def match(line, keyword):
     """ If the first part of line (modulo blanks) matches keyword,
-    returns the end of that line. Otherwise returns None"""
+    returns the end of that line. Otherwise checks if keyword is
+    anywhere in the line and returns that section, else returns None"""
 
     line = line.lstrip()
     length = len(keyword)
     if line[:length] == keyword:
         return line[length:]
     else:
-        return None
+        if keyword in line:
+            return line[line.index(keyword):]
```

Comment:
since signal Level isn't at the beginning of the line, look for it within the line
>                    Quality=98/100  Signal level=63/100